.eslintrc

```diff
@@ -0,0 +1,21 @@
+{
+  env: {
+    es6: true,
+    browser: true
+  },
+  extends: ["eslint:recommended", "standard"],
+  rules: {
+    "no-console": [2, { allow: ["warn", "error"] }],
```

Comment:
This is to make sure we don't leave any console.log statements in our production code (very easy to add a bunch of console.log statements while debugging and then miss one). The statements won't be removed by Prettier automatically so we can still use console.logs during development. Before we commit our code, we can just run our linter (NPM run lint) and it will warn us to remove those statements and show us where they are.